test/index.js

```diff
@@ -18,6 +18,9 @@ series([
     console.log('Running npm install in fixtures/basic...')
     exec('npm install', {cwd: util.fixtureSubdir('basic')}, cb)
   }, function (cb) {
+    console.log('Running npm install in fixtures/basic-renamed-to-electron...')
+    exec('npm install', {cwd: util.fixtureSubdir('basic-renamed-to-electron')}, cb)
+  }, function (cb) {
```

Comment:
Since we keep adding these, might as well genericize it.

_(Note to self: add optional part about adding `npm install` to test fixtures docs)_